From: Benda Xu <heroxbd@gentoo.org>
Date: Wed, 29 Oct 2014 10:06:53 +0900
Subject: disable the tests replying on permission bits exceptions

Bug: https://code.google.com/p/casacore/issues/detail?id=73
Subject: disable the tests on file/directory operations

They are tBucketFile, tByteIO, tDirectory, tFile, tRegularFile, tSymLink,
resulting in failure if tested by superuser.

For example, from casa/OS/test/tSymlink.cc

    file.setPermissions (0444);
    if (doExcp) {
        try {
            SymLink symLink1(Path("tSymLink_tmp/isDir/newB"));
        }
        catch (AipsError x) {
            cout << x.getMesg () << endl;                    // cannot create
        }
        try {
            newLink2.create("a");
        }
        catch (AipsError x) {
            cout << x.getMesg () << endl;                    // cannot create
        }
    }
---
 casa/IO/test/CMakeLists.txt | 2 --
 casa/OS/test/CMakeLists.txt | 4 ----
 2 files changed, 6 deletions(-)

diff --git a/casa/IO/test/CMakeLists.txt b/casa/IO/test/CMakeLists.txt
index 188a6c4..6dafd38 100644
--- a/casa/IO/test/CMakeLists.txt
+++ b/casa/IO/test/CMakeLists.txt
@@ -3,9 +3,7 @@ tAipsIOCarray
 tAipsIO
 tBucketBuffered
 tBucketCache
-tBucketFile
 tBucketMapped
-tByteIO
 tByteSink
 tByteSinkSource
 tFilebufIO
diff --git a/casa/OS/test/CMakeLists.txt b/casa/OS/test/CMakeLists.txt
index 520ceed..328bd9f 100644
--- a/casa/OS/test/CMakeLists.txt
+++ b/casa/OS/test/CMakeLists.txt
@@ -3,10 +3,8 @@ tCanonicalConversion
 tConversion
 tConversionPerf
 tDataConversion
-tDirectory
 tDirectoryIterator
 tEnvVar
-tFile
 tHostInfo
 tIBMConversion
 tLECanonicalConversion
@@ -16,8 +14,6 @@ tModcompConversion
 tMutex
 tPath
 tPrecTimer
-tRegularFile
-tSymLink
 tTime
 tTimer
 tVAXConversion
